libs/platform/src/lib/components/value-help-dialog/value-help-dialog/value-help-dialog.component.html

```diff
@@ -0,0 +1,341 @@
+                                            (onCloseClick)="removeExcluded(excludedIndex)"
+                                            *ngFor="let item of validExcludedItems; index as excludedIndex; trackBy:_trackByConditionFn"
+                                        >
+                                            <ng-container [ngTemplateOutlet]="conditionValue" [ngTemplateOutletContext]="{ item: item }"></ng-container>
+                                        </fd-token>
+                                    </fd-toolbar>
+                                    <button
+                                        (click)="clearExcludedItems()"
+                                        class="fdp-value-help-dialog__tokens-clear"
+                                        fd-button
+                                        fdType="transparent"
+                                        glyph="sys-cancel"
+                                        i18n-aria-label="@@platformI18nValueHelpDialog.ClearExcludedAriaLabel"
+                                        aria-label="clear excluded items"></button>
+                                </div>
+                            </div>
+                        </ng-container>
+                    </div>
+                </div>
+                <div class="fdp-value-help-dialog__actions-row">
+                    <button
+                        fd-button
+                        data-go-advanced
+                        i18n-label="@@platformI18nValueHelpDialog.SearchButton"
+                        label="Go"
+                        fdType="emphasized"
+                        *ngIf="mobile && isOpenAdvanced"
+                        [compact]="true"
+                        (click)="searchAdvanced()"
+                    ></button>
+                    <button
+                        fd-button
+                        data-succes
+                        i18n-label="@@platformI18nValueHelpDialog.SuccessButton"
+                        label="OK"
+                        fdType="emphasized"
+                        *ngIf="selectedTab !== _tabTypes.advancedSearch"
+                        [compact]="true"
+                        (click)="success()"></button>
+                    <button
+                        fd-button
+                        data-dismiss-dialog
+                        fd-dialog-decisive-button
+                        i18n-label="@@platformI18nValueHelpDialog.CancelButton"
+                        label="Cancel"
+                        fdType="transparent"
+                        [compact]="true"
+                        (click)="dismiss()"></button>
+                </div>
+            </ng-template>
+        </fd-dialog-footer>
+    </fd-dialog>
+</ng-template>
+
+<ng-template #selectedPanelTitle>
+    <h6 fd-title><ng-container i18n="@@platformI18nValueHelpDialog.Selected.IncludedLabel">Selected items</ng-container> ({{ selectedAndIncluded }})</h6>
```

Comment:
I would recommend to keep i18n marker wrapping whole place including variable as well, it will gave a translator a better context